src/components/posts/helperfunctions.js

```diff
@@ -0,0 +1,22 @@
+import { differenceInHours, differenceInDays } from 'date-fns'
```

Comment:
The tests are failing because this library can't be found - do you need to commit a package.json update?

Reply:
I expect so - first time adding a library to the project. So date-fns needs to be added as a node-module I take it.

Reply:
Yes, if you do npm install it should be added to the `package.json` automatically?

Reply:
npm install followed by git status gives 

'Your branch is up to date with 'origin/15-view-posts-client'.

nothing to commit, working tree clean'